extensions/amp-form/amp-form.md

```diff
@@ -234,7 +234,7 @@ The response is expected to be a valid JSON Object. For example, if the publishe
 
 Both success and error responses should have a `Content-Type: application/json` header. `submit-success` will render for all responses that has a status of `2XX`, all other statuses will render `submit-error`.
 
-Publishers can render these in a template inside their forms as follows.
+Publishers can render these in a inlined template inside their forms as follows.
```

Comment:
`an inlined` 

Thanks for chaining this, I prefer `inline` to reduce confusion with `nested` templates that we have been chatting about

Reply:
Done